librz/include/rz_analysis.h

```diff
@@ -471,13 +472,13 @@ enum {
 	RZ_ANALYSIS_REFLINE_TYPE_MIDDLE_AFTER = 8 /* as above but for lines after disasm */
 };
 
-enum {
+typedef enum {
 	RZ_ANALYSIS_RET_NOP = 0,
-	RZ_ANALYSIS_RET_ERROR = -1,
-	RZ_ANALYSIS_RET_DUP = -2,
-	RZ_ANALYSIS_RET_NEW = -3,
-	RZ_ANALYSIS_RET_END = -4
-};
+	RZ_ANALYSIS_RET_ERROR = -1, // Basic block ended because of analysis error.
```

Comment:
@ret2libc @thestr4ng3r @wargio opinions on this change? I personally have nothing against it, for the record.

Reply:
i think is ok